packages/truss/src/methods.ts

```diff
@@ -38,17 +40,19 @@ export function newParamMethod(abbr: UtilityName, prop: keyof Properties): Utili
  * @param defs a map of abbreviation name --> value
  * @param baseName the base name to use, i.e. `mt`
  * @param includePx generate an extra `${baseName}Px` method that calls the base method with a converted px value
+ * @param baseDefs additional properties to set for the base method
  */
 export function newMethodsForProp<P extends Prop>(
   prop: P,
-  defs: Record<UtilityName, Properties[P]>,
+  defs: Record<UtilityName, Properties[P] | Properties>,
   baseName: string | null = prop,
   includePx: boolean = false,
+  baseDefs?: Omit<Properties, P>,
```

Comment:
Suggestion: Maybe call this `valueMethodExtraProperties`? 

I think the "value method" prefix would give me the best hint about remembering what this does :-)